modules/microshift-install-rhel-image-mode-run-container.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-container_{context}"]
+= Running the {microshift-short} bootc container
+
+If the host is already configured to have a `rhel` volume group with free space, this configuration is inherited by the container. It can be used by the {microshift-short} LVMS Container Storage Interface (CSI) to allocate storage.
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have `sudo` permissions.
+* You have a pull secret file for downloading the required {microshift-short} container images.
+* You configured host container storage for reusing available container images.
+
+.Procedure
+
+. Run the container for the {microshift-short} service by entering the following command:
++
+[source,terminal]
+----
+PULL_SECRET=~/.pull-secret.json
+IMAGE_NAME=microshift-4.16-bootc
+
+$ sudo podman run --rm -it --privileged \
+    -v "${PULL_SECRET}":/etc/crio/openshift-pull-secret:ro \
+    -v /var/lib/containers/storage:/var/lib/containers/storage \
+    --name "${IMAGE_NAME}" \
+    "${IMAGE_NAME}"
+----
+//moved the hotwiring openvswitch over to CNI module
+
+[NOTE]
+====
+The `systemd-modules-load` service fails to start in the container if the host kernel version is different from the bootc image kernel version. This failure can be safely ignored as all the necessary kernel modules have been loaded by the host.
+====
+
+A login prompt like the following is presented in the terminal after {microshift-short} has started:
++
+[source,terminal]
+----
+$ tbd example
+----
+
+. Log into the running container using the `redhat:<password>` credentials.
++
+[source,terminal]
+----
+$ tbd example
+----
+
+. Verify that all the {microshift-short} pods are up and running without errors by running the following command:
++
+[source,terminal]
+----
+$ watch sudo oc get pods -A \
+    --kubeconfig /var/lib/microshift/resources/kubeadmin/kubeconfig
+----
+.Example output
+[source,text]
+----
+Run the sudo shutdown now command to stop the container.
```

Comment:
The `sudo shutdown now` is a command to be run within the container to shut it down. 
If we mention this, we should do it in the end and as a separate section.

Reply:
Oh, I see. Do we have an output from a cluster? Does it look like any other MicroShift example? I added the vanilla content from our usual snippet. WDYT?